alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -651,7 +651,7 @@ private function initMembers()
 		//Get bundle configuration stored in conf_vars
 		$confVars = $this->uiConf->getConfVars();
 		if (!$confVars) {
-			KExternalErrors::dieGracefully("Missing bundle configuration in uiConf, uiConfID: $this->uiconfId");
+			KExternalErrors::dieError("Missing bundle configuration in uiConf, uiConfID: $this->uiconfId");
```

Comment:
please add the error ENUM `KExternalErrors::BUNDLE_CREATION_FAILED`

Reply:
done